renderdoc/driver/vulkan/vk_acceleration_structure.cpp

```diff
@@ -766,74 +866,142 @@ void VulkanAccelerationStructureManager::Apply(ResourceId id, const VkInitialCon
   }
 }
 
-VulkanAccelerationStructureManager::Allocation VulkanAccelerationStructureManager::CreateReadBackMemory(
-    VkDevice device, VkDeviceSize size, VkDeviceSize alignment)
+GPUBuffer VulkanAccelerationStructureManager::CreateReadBackMemory(VkDevice device, VkDeviceSize size)
 {
-  VkBufferCreateInfo bufInfo = {
+  GPUBuffer result;
+  result.Create(m_pDriver, device, size, 1,
+                GPUBuffer::eGPUBufferReadback | GPUBuffer::eGPUBufferAddressable);
+
+  m_pDriver->GetResourceManager()->SetInternalResource(GetResID(result.mem));
+  m_pDriver->GetResourceManager()->SetInternalResource(GetResID(result.buf));
+
+  return result;
+}
+
+VulkanAccelerationStructureManager::Allocation VulkanAccelerationStructureManager::CreateReplayMemory(
```

Comment:
I don't mind leaving this as a helper but I think this function (honestly both functions) would be better named as `CreateReplayBuffer` or `CreateTempReplayBuffer` as it's not just allocating memory.